.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+    -   id: trailing-whitespace
+        exclude: |
+            (?x)^(
+                README.md|
+                docs/
+            )$
+    -   id: end-of-file-fixer
+        exclude_types: [html]
```

Comment:
As per discussion, we ignore html files here. They won't get a trailing newline.